lib/models/downloadstatus.dart

```diff
@@ -0,0 +1,8 @@
+
+// ignore_for_file: constant_identifier_names
+
+enum DownloadStatus {
```

Comment:
**suggestion**: we follow Dart's best coding practices so we don't ignore these rules. We always use lower camel case for enuma